File: easy-solution/minesp3164/python/1295.py
class Solution(object):
    def findNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        count = 0
        for i in range(len(nums)):
            digit = len(str(nums[i]))
            if digit % 2 == 0:
                count = count + 1
        return count

File: easy-solution/bear-bear-bear/1295.rs
impl Solution {
    pub fn find_numbers(nums: Vec<i32>) -> i32 {
        let mut count: i32 = 0;
        for num in &nums {
            if num.to_string().len() % 2 == 0 {
                count += 1;
            }
        }
        count
    }
}
